routers/web/admin/admin.go

```diff
@@ -133,12 +135,21 @@ func DashboardPost(ctx *context.Context) {
 
 	// Run operation.
 	if form.Op != "" {
-		task := cron.GetTask(form.Op)
-		if task != nil {
-			go task.RunWithUser(ctx.Doer, nil)
-			ctx.Flash.Success(ctx.Tr("admin.dashboard.task.started", ctx.Tr("admin.dashboard."+form.Op)))
-		} else {
-			ctx.Flash.Error(ctx.Tr("admin.dashboard.task.unknown", form.Op))
+		switch form.Op {
+		case "sync_repo_sync":
+			go func() {
```

Comment:
Not ideal. The context is for an HTTP request, and it could be canceled after the request finished. Maybe `graceful.GetManager().HammerContext()`

Reply:
Wait, the ctx is unused, see the comment below.